README.md

```diff
@@ -14,14 +14,18 @@ If you see a step below that could be improved (or is outdated), please update t
 ### Prerequisite
 
 1. Make sure you have Node.js version >= 10.
-
-- We recommend using [nvm](https://github.com/creationix/nvm): `nvm use`.
```

Comment:
`nvm use` read the `.nvmrc` version, it's great

Reply:
Niiice!